<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoDrawer</class>
 <widget class="QMainWindow" name="AutoDrawer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoDrawer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../AutoDrawer v1/Assets/Icon.ico</normaloff>../../AutoDrawer v1/Assets/Icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:transparent</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(44, 48, 46);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="ImageOpt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>490</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 55, 53);
border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="ScaleFrame">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 55, 53);
border-radius: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSlider" name="ScaleSlider">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>335</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">        QPushButton {
            color: white;
        }
        QPushButton:hover {
            color: grey;
        }
</string>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="ScaleText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-1</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="scaleNumber">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>8</y>
         <width>51</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(49, 49, 49);
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="HeightText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>235</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Sans Serif&quot;;
background:transparent;</string>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="WidthText">
      <property name="geometry">
       <rect>
        <x>246</x>
        <y>45</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Sans Serif&quot;;
background:transparent;</string>
      </property>
      <property name="text">
       <string>Width</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="heightBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>234</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(44, 48, 46);
border-top-left-radius: 10px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 10px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="widthBox">
      <property name="geometry">
       <rect>
        <x>247</x>
        <y>80</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(44, 48, 46);
border-top-left-radius: 0px;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 0px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro Black</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">        QPushButton {
            color: white;
        }
        QPushButton:hover {
            color: grey;
        }
</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Minimize">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro Black</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">        QPushButton {
            color: white;
        }
        QPushButton:hover {
            color: grey;
        }
</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="Logo">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;logo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Logo_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Autodrawer v2.0-alpha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>250</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgb(58,61,59);
border-radius: 10px;
}
        QPushButton:hover {
background: rgb(55,55,55);
        }</string>
     </property>
     <property name="text">
      <string>Custom Pattern</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>109</y>
       <width>171</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgb(58,61,59);
border-radius: 0px;
}
        QPushButton:hover {
background: rgb(55,55,55);
        }</string>
     </property>
     <property name="text">
      <string>Load Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>155</y>
       <width>171</width>
       <height>42</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgb(58,61,59);
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
}
        QPushButton:hover {
background: rgb(55,55,55);
        }</string>
     </property>
     <property name="text">
      <string>Save Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>60</y>
       <width>46</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgb(58,61,59);
border-top-left-radius: 0px;
border-top-right-radius: 10px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;
}
        QPushButton:hover {
background: rgb(55,55,55);
        }</string>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
    <widget class="QLabel" name="Logo_3">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>210</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Drawing Pattern&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>460</y>
       <width>179</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgb(58,61,59);
border-top-left-radius: 10px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;
}
        QPushButton:hover {
background: rgb(55,55,55);
        }</string>
     </property>
     <property name="text">
      <string>Upload Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>697</x>
       <y>460</y>
       <width>179</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgb(58,61,59);
border-top-left-radius: 0px;
border-top-right-radius: 10px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;
}
        QPushButton:hover {
background: rgb(55,55,55);
        }</string>
     </property>
     <property name="text">
      <string>Clear Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>512</y>
       <width>361</width>
       <height>44</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">        QPushButton {
background: rgb(58,61,59);
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;
        }
        QPushButton:hover {
background: rgb(55,55,55);
        }
</string>
     </property>
     <property name="text">
      <string>Process Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>560</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgb(58,61,59);
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
}
        QPushButton:hover {
background: rgb(55,55,55);
        }</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="intervalTextBox">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>340</y>
       <width>179</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(56, 56, 56);
border-top-left-radius: 10px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 10px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="clickDelayTextBox">
     <property name="geometry">
      <rect>
       <x>697</x>
       <y>340</y>
       <width>179</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(56, 56, 56);
border-top-left-radius: 0px;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 0px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Logo_4">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>310</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Interval&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Logo_5">
     <property name="geometry">
      <rect>
       <x>695</x>
       <y>310</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Click Delay&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Logo_6">
     <property name="geometry">
      <rect>
       <x>695</x>
       <y>380</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Transparency Threshold&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Logo_7">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>380</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Black Threshold&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>705</x>
       <y>60</y>
       <width>171</width>
       <height>113</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(58,61,59);
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;</string>
     </property>
     <property name="movement">
      <enum>QListView::Snap</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>705</x>
       <y>175</y>
       <width>143</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(58,61,59);
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Set Directory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>175</y>
       <width>26</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(58,61,59);
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 0px;</string>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QListView" name="drawingList">
     <property name="geometry">
      <rect>
       <x>705</x>
       <y>220</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(58,61,59);
border-radius: 10px;</string>
     </property>
     <property name="movement">
      <enum>QListView::Snap</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="transThresh">
     <property name="geometry">
      <rect>
       <x>697</x>
       <y>410</y>
       <width>179</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(56, 56, 56);
border-top-left-radius: 0px;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 0px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="blackThresh">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>410</y>
       <width>179</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(56, 56, 56);
border-top-left-radius: 10px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 10px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ImageFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58, 61, 59);
border-radius: 5px;
background-size: cover;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>60</y>
       <width>122</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgb(58,61,59);
border-top-left-radius: 10px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;

}
        QPushButton:hover {
background: rgb(55,55,55);
        }</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
